src/misc/meson.build

```diff
@@ -3,6 +3,7 @@ libmisc_sources = [
   'fs_utils_posix.cpp',
   'fs_utils_win32.cpp',
   'messages.cpp',
+  'pacer.cpp',
```

Comment:
Pacer can be added to the visual studio solution files using:
`git grep soft_limiter vs/`

Which will show you the files that need to be edited. 

Just do it exactly like the limiter, place in those xml blocks (in alphabetical order), and the CI windows workflow should build and link without issues.